notes.md

```diff
@@ -90,16 +90,40 @@ Matroska provides several methods to link two or many `Segment Elements` togethe
 
 ## Hard Linking
```

Comment:
The description of the patch doesn't mention it changes the a MUST into a SHOULD. It must say so and why.